zabbix_auto_config/processing.py

```diff
@@ -369,6 +369,10 @@ def __init__(self, name, db_uri, zabbix_config):
         self.dryrun = zabbix_config["dryrun"]
         self.failsafe = zabbix_config["failsafe"]
         self.tags_prefix = zabbix_config["tags_prefix"]
+        if "managed_inventories" in zabbix_config:
+            self.managed_inventories = zabbix_config["managed_inventories"].split(",")
```

Comment:
You might want to strip the strings collected, or you could get property keys with whitespace

Reply:
thanks for pointing out this, it will be fixed.